CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing to Oboe.js
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to Oboe.js. These are just guidelines, not rules, use your best judgment and feel free to propose changes to this document in a pull request.
+
+## Read the docs :)
+
+Oboe.js has some [awesome documentation](http://oboejs.com/) explaining how and why to use the library.
+
+## Questions/Help
+
+Sometimes your question can be addressed by reading the [API](http://oboejs.com/api) closely. It's short and nicely organized! 
+
+Please post questions to [StackOverflow](http://stackoverflow.com/) using the `javascript` tag and mention Oboe.js in the title.
+
+If you file an issue with a question, it will be closed. We're not trying to be mean, it just helps keep the issues tab cleaner so we can keep improving the library.
+
+## Reporting Bugs / Requesting Features
+
+If you've found an issue, please submit it in [the issues](https://github.com/jimhigson/oboe.js/issues). A link to a [jsbin](https://jsbin.com/)/[codepen](http://codepen.io/)/[plunkr](https://plnkr.co/) that demonstrates the issue if it's on the client, or a github repo if it's on the server, greatly increases our ability to help.
+
+## Pull Requests
+
+If you would like to add functionality, please submit [an issue](https://github.com/jimhigson/oboe.js/issues)
+first to make sure it's a direction we want to take.
+
+Please do the following:
+* Follow the existing styles
+* Create an example for that demonstrates your changes so people can see how your changes work
+
+In your PR description include any information that will help a maintainer understand and test your changes. The easier it is to read and run your PR, the faster it can get merged!
+
+#### Notes
+
+- Please don't commit any changes to the `dist/` directory. This is only committed for releases.
```

Comment:
I copied this over because it seemed reasonable that `dist` is built as part of preparing for a release, but it seems like the normal development process leads to you changing `dist`, as mentioned by @charlierudolph in #87.

@jimhigson Would you expect all commits to change `dist`?